cds-plugin.js

```diff
@@ -12,10 +12,7 @@ if (cds.env.cov2ap.plugin) {
   cds.on("bootstrap", async (app) => {
     app.use(cov2ap.singleton());
   });
-  if (cds.env.cov2ap.build && cds.build && cds.build.register) {
-    cds.build.register("cov2ap", {
```

Comment:
I already us register, what's the difference?

Reply:
The old `register` version using the `impl` property is deprecated and no longer supported. I planned to delete it with cds-dk 8

Reply:
OK. Got it. I tested it and it works for me...